Sources/App/Commands/Analyzer.swift

```diff
@@ -123,11 +123,11 @@ func pullOrClone(application: Application, package: Package) throws -> EventLoop
             try Current.shell.run(command: .init(string: "git reset --hard"), at: cacheDir)
             let branch = package.repository?.defaultBranch ?? "master"
             try Current.shell.run(command: .gitCheckout(branch: branch), at: cacheDir)
-            try Current.shell.run(command: .gitPull(allowingPrompt: false), at: cacheDir)
+            try Current.shell.run(command: .gitPull(), at: cacheDir)
```

Comment:
John has also merge this change upstream now, so we can easily bring it back if needed.